src/db/util/sequelize-model.ts

```diff
@@ -182,12 +183,27 @@ export const defineSequelizeModel =
       });
     };
 
```

Comment:
We will need to add a new type function definition in order to do this approach.
```suggestion
export type DestroySequelizeFn<F extends FieldDefinition> = {
  (args: { where: WhereCond<F>; trx?: Knex.Transaction<any, any> }): Promise<number>;
  (args: { values: Partial<UserDataOf<F>>; where: WhereCond<F>; trx?: Knex.Transaction<any, any> }): Promise<number>;
};
```

Reply:
It may not be needed to`export` this type function. It's an 'extra' from copy-pasting